src/pages/admin/regulatory/forms/CreateRegulatoryForm.js

```diff
@@ -127,7 +127,7 @@ const CreateRegulatoryForm = props => {
               <br />
               <FieldArray
                 name = "attachments"
-                render= {arrayaHlpers => (
+                render= {arrayHelpers => (
```

Comment:
Try to stick with the correct naming conventions. Try to avoid spelling mistakes